content/ember/v5/deprecate-template-action.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Action helper and modifier 
+until: 6.0.0
+since: 5.10.0
```

Comment:
Just double checking, is this deprecation targeting 5.10.0? (so not the next version)

Reply:
depends on if we merge the implementation before 5.9 becomes beta?

Reply:
Alright, just wanted to double check this. Sounds good then!